model/testresult/testresult.go

```diff
@@ -0,0 +1,81 @@
+package testresult
+
+import (
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/db/bsonutil"
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const (
+	collection = "testresults"
+)
+
+// TestResult contains test data for a task.
+type TestResult struct {
+	ID        bson.ObjectId `bson:"_id" json:"_id"`
```

Comment:
should probably have the json version be "id" to be consistent with the other models

Reply:
done